### DISCUSSION TITLE
Clicking on mapper name opens my own user profile and not the mapper's

### DISCUSSION THREAD

**Discussion by ikkue** [ppy/osu#15474] [Q&A] [answered]
As stated in the title, when I click on **the mapper's name** on the song card in the **song select** menu, the profile card for **my user profile pops up** instead of the mapper's. I don't know if this is a known bug or an intentional thing (which would be very weird), but I'm reporting here just to make sure anyways!

https://user-images.githubusercontent.com/11213135/140401533-1ebb26f2-3b7f-4ebc-a8b0-731f7f2db91a.mp4

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/15033
https://github.com/ppy/osu/issues/5966
